typescript/web/src/components/export-button/export-modal/modal-header.tsx

```diff
@@ -24,27 +23,14 @@ export const ModalHeader = () => {
           Your dataset contains {imagesNumber} images and {labelsNumber} labels.
         </Text>
       </Skeleton>
-      <Box display="inline-block" width="100%" pl={8} pr={8} pt={4}>
-        {
-          // eslint-disable-next-line eqeqeq
-          numberUndefinedLabelsOfDataset != null &&
-            numberUndefinedLabelsOfDataset !== 0 && (
-              <Alert status="warning" borderRadius={5}>
-                <AlertIcon />
-                <AlertTitle mr={2} fontSize={15} whiteSpace="nowrap">
-                  Missing Class
-                </AlertTitle>
-                <AlertDescription fontSize={13} fontWeight="medium">
-                  {`${numberUndefinedLabelsOfDataset} ${
-                    numberUndefinedLabelsOfDataset === 1 ? "label" : "labels"
-                  } ${
-                    numberUndefinedLabelsOfDataset === 1 ? "has" : "have"
-                  } no class assigned. Only labels with a class
-            are exported.`}
-                </AlertDescription>
-              </Alert>
-            )
-        }
+      <Box width="100%" pl={8} pr={8} pt={4}>
+        <Alert status="info" borderRadius={5}>
```

Comment:
While refactoring it, I'd have taken the opportunity to move this block in it's own `Disclaimer` component (_or `LabelWithoutLabelClassDisclaimer` if needed_).

Reply:
Fixed